addons/web/static/src/scss/bootstrap_review_frontend.scss

```diff
@@ -236,3 +236,37 @@ $-color-for-gray-200-bg: adjust-color-to-background($body-color, $gray-200);
         color: adjust-color-to-background($text-muted, $dropdown-bg, mute-color($color-contrast-light), mute-color($color-contrast-dark)) !important;
     }
 }
+
+// The following code adapts the style of disabled inputs to maintain color
+// consistency and make them more recognizable.
+.form-select {
+    // This is necessary to achieve a consistent "border rendering behaviour"
+    // across form-select and form-control when using semi-transparent
+    // borders and a background color (eg. disabled inputs).
+    // Otherwise, the color rendering of the semi-transparent border may be altered
+    // (the color may be darker depending on the background color used).
+    background-clip: padding-box;
+
+    &:disabled:not([multiple]):not([size]:not([size="1"])) {
```

Comment:
The `:not([size]:not([size="1"]))` part gives me a headache ^^ It this overriding a specific bootstrap rule? If not maybe `:where(:not([size]), [size="1"])` would be more readable and less forcing ?

Reply:
`:where(:not([size]), [size="1"])` works well, thanks ! 👍